test/integration/user_can_see_past_orders_test.rb

```diff
@@ -0,0 +1,38 @@
+class UserCanSeePastOrdersTest < ActionDispatch::IntegrationTest
+  test "user can see past orders in" do
+    create_and_login_user
+    user = User.first
+
+    order1 = user.orders.create(total: 1001,
+                                created_at: Time.new(2011, 11, 10, 15, 25, 0))
```

Comment:
Do not use `Time.new` without zone. Use one of `Time.zone.local`, `Time.new.current`, `Time.new.in_time_zone`, `Time.new.utc`, `Time.new.getlocal`, `Time.new.iso8601`, `Time.new.jisx0301`, `Time.new.rfc3339`, `Time.new.to_i`, `Time.new.to_f` instead.